src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -168,7 +168,6 @@ internal static class IDEDiagnosticIds
         public const string InvokeDelegateWithConditionalAccessId = "IDE1005";
         public const string NamingRuleId = "IDE1006";
         public const string UnboundIdentifierId = "IDE1007";
-        public const string UnboundConstructorId = "IDE1008";
```

Comment:
UnboundIdentifierId still needs to go too, but that's harder and we need compiler support here for that.